### ISSUE TITLE
Excellent project, sad abandonment. RIP

### ISSUE THREAD

**Issue by marcorroma** [levelten/ga-in#10]
It appears this plugin has been abandoned.
I'm so sorry, it was my favorite, but I'll be installing another one shortly to integrate with Google Analytics 4.
RIP

**Comment by mrleemon** [comment]
Is there an alternative plugin similar to this one?

**Comment by Aetles** [comment]
I have not found a good alternative. The official one from Google (Site Kit) currently only display the metrics for administrators (see https://github.com/google/site-kit-wp/issues/852) and lack support for GA4 (although it's coming, see https://github.com/google/site-kit-wp/issues/2220).

**Comment by Hazzeldorn** [comment]
When creating a new Google Analytics Property, you can still create a Universal Analytics Property in addition to the new V4 Property. This way, the plugin can still be used.

Alternatively, use this fork: https://wordpress.org/plugins/metrics-query/

**Comment by Aetles** [comment]
Yes, but there are other issues as well, with PHP errors and a lot of reports of problems [in the support forum](https://wordpress.org/support/plugin/ga-in/).

**Comment by superbmario** [comment]
https://wordpress.org/plugins/analytics-insights/ is now a great alternative. Created by the original developer of the Google Analytics Dashboard for WordPress plugin (which was sold to ExactMetrics a few years ago and ruined)

**Comment by cbratschi** [comment]
@superbmario GAinWP has some additional features. I consider switching too but I am not sure the migration is that easy. Custom metrics APIs are equal and share the same quote escaping bug. Not sure if AIWP still supports UA codes.

**Comment by marcorroma** [comment]
I'm using the "Metrics Query" plugin, if that helps
